src/main/java/redis/clients/jedis/Commands.java

```diff
@@ -13,7 +13,7 @@ public void set(final String key, final String value, final String nxxx, final S
 
   public void get(final String key);
 
-  public void exists(final String key);
+  public void exists(final String... keys);
```

Comment:
@charsyam 
We're breaking binary compatibility now. 
Could you leave origin method as is but adding @Deprecated?